policy/executor/internal/nodes.go

```diff
@@ -530,22 +530,27 @@ func (nodeData *ReportingJobNodeData) score() (*policy.Score, error) {
 		return s, nil
 	}
 
-	calculator, err := policy.NewScoreCalculator(nodeData.scoringSystem)
-	if err != nil {
-		return nil, err
+	s := &policy.Score{
+		Type: policy.ScoreType_Unscored,
```

Comment:
Do we need to fill in the other props, such as `datacompletion` and `datatotal` here?

Reply:
Good catch. This lead to an even simpler solution. We can re-use the avg calculator. That also takes care of the rest.